test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorPageTest.cs

```diff
@@ -497,6 +514,28 @@ public void IsSectionDefined_ThrowsIfPreviousSectionWritersIsNotRegistered()
         }
 
         [Fact]
+        public async Task EnsureRenderedBodyOrSections_SucceedsIfDefinedSectionsAreNotRendered_AndIgnored()
```

Comment:
Isn't `EnsureRenderedBodyOrSections_SucceedsIfRenderBodyIsNotCalled_ButAllDefinedSectionsAreRendered` already checks this?